testify/test_program.py

```diff
@@ -86,7 +86,13 @@ def default_parser():
 
     parser.set_defaults(verbosity=test_logger.VERBOSITY_NORMAL)
     parser.add_option("-s", "--silent", action="store_const", const=test_logger.VERBOSITY_SILENT, dest="verbosity")
```

Comment:
afaict, we're never respecting the `--silent` option. if that's the case, should we fix that too?

Reply:
err i guess it is being respected in `test_logger` for other output suppression https://github.com/Yelp/Testify/blob/master/testify/test_logger.py#L175

Reply:
That is a good question that I have considered, but couldn't think of a simple option to implement.

In my opinion, if I tell a test framework to be quiet, I only want it to print a count of tests that ran, and which failed. Everything else should not be printed out, including prints from the code that we are running, however that may require patching something that could mess with the results.